basics/movie_schedule.py

```diff
@@ -0,0 +1,15 @@
+current_movies = {'The Grinch': '11:00am',
+                  'Rudolph': '1:00pm',
+                  'Frosty the Snowman': '3:00pm',
+                  'Christmas Vacation': '5:00pm'}
+
+print("We're showing the following movies:")
+for key in current_movies:
+    print(key)
+movie = input('What movie would you like the showtime for?\n')
+
+showtime = current_movies.get(movie)
+if showtime == None:
```

Comment:
can we add an elif statement like if the selected movie is not in the list it should ask to select another like recursion. until the movie selected in the list